lib/Config.php

```diff
@@ -26,6 +43,15 @@ public function __construct(array $nameservers, array $knownHosts = [], int $tim
         if ($attempts < 1) {
             throw new ConfigException("Invalid attempt count ({$attempts}), must be 1 or greater");
         }
+        if ($ndots > 1 && \count($searchList) === 0) {
+            throw new ConfigException("Invalid ndots count ({$ndots}), must be 1 if search list is empty");
```

Comment:
If it must be `1`, we should probably check for `!== 1` here instead of `> 1`?

Reply:
Removed, works same with a default `= 0`